### PR TITLE
added Petio

### PR DESCRIPTION
**Pull request by oldhunterr**
## Description
- New Script Petio  ct (Plex Requester) created as a request from discussion : 
https://github.com/tteck/Proxmox/discussions/3136#discussion-6780767

Fixes # (issue)
- None

## Type of change
- New Script

Please delete options that are not relevant.

- [x] New Script (Develop a new script or set of scripts that are fully functional and thoroughly tested)
- [x] I have performed a self-review of my code, adhering to established codebase patterns and conventions.

### CODE REVIEW

File: ct/petio.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/oldhunterr/Proxmox/main/misc/build.func)
```
Comment (tteck) [inline ct/petio.sh]: Wrong source

File: ct/petio.sh [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# License: MIT
+# https://github.com/oldhunterr/Proxmox/raw/main/LICENSE
```
Comment (tteck) [inline ct/petio.sh]: Wrong license link

File: install/petio-install.sh [hunk 3]
```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# License: MIT
+# https://github.com/oldhunterr/Proxmox/raw/main/LICENSE
```
Comment (tteck) [inline install/petio-install.sh]: Wrong license link

File: install/petio-install.sh [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+Type=simple
+Restart=on-failure
+RestartSec=1
+ExecStart=/opt/Petio/bin/petio-linux
+User=petio
+
+[Install]
+WantedBy=multi-user.target
+EOF
+systemctl daemon-reload
```
Comment (tteck) [inline install/petio-install.sh]: Not needed

Comment (oldhunterr) [reply install/petio-install.sh]: Thank you for the review.

Are you referring to these three lines of code as not needed?

```shell
WantedBy=multi-user.target
EOF
systemctl daemon-reload
```

Please let me know if there's anything else required or if I missed something.

Thanks!

Comment (tteck) [reply install/petio-install.sh]: This `systemctl daemon-reload` is not needed

Comment (oldhunterr) [reply install/petio-install.sh]: Removed

File: install/petio-install.sh [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+sed -i 's/bindIp: 127.0.0.1/bindIp: 0.0.0.0/' /etc/mongod.conf
+systemctl enable -q --now mongod.service
+msg_ok "MongoDB 4.4 Installed"
+
+
+msg_info "Installing Petio"
+useradd -M --shell=/bin/false petio
+mkdir /opt/Petio
+wget https://petio.tv/releases/latest -O petio-latest.zip
+unzip petio-latest.zip -d /opt/Petio
```
Comment (tteck) [inline install/petio-install.sh]: is `wget https://petio.tv/releases/latest -O petio-latest.zip` and `unzip petio-latest.zip -d /opt/Petio` silent? Also you have left the `petio-latest.zip` behind.

Maybe 
```
wget -q https://petio.tv/releases/latest -O petio-latest.zip
$STD unzip petio-latest.zip -d /opt/Petio
rm -rf petio-latest.zip
```
to make silent and remove `petio-latest.zip`. Have you tested this script?

**Comment by tteck** [comment]
Thanks

**Pull request merged**